flutter_frontend_server/test/to_string_test.dart

```diff
@@ -45,7 +46,7 @@ Future<void> main(List<String> args) async {
     final ProcessResult runResult = Process.runSync(dart, <String>[regularDill]);
     _checkProcessResult(runResult);
     String paintString = '"Paint.toString":"Paint(Color(0xffffffff))"';
-    if (const bool.fromEnvironment('dart.vm.product', defaultValue: false)) {
+    if (buildDir.contains('release')) {
```

Comment:
Checking `dart.vm.product` no longer works because the tests are running in a `release` Dart VM when using a prebuilt Dart SDK. Checking whether the build output directory contains 'release' gives the right signal whether or not the prebuilt Dart SDK is used.